### PR TITLE
[Docs] add language to geopartitioning code blocks

### PR DESCRIPTION
**Pull request by polarweasel**
Preview [is here](https://deploy-preview-7795--infallible-bardeen-164bc9.netlify.app/latest/explore/multi-region-deployments/row-level-geo-partitioning)

### CODE REVIEW

**Comment by polarweasel** [comment]
@deeps1991 volunteered to update the broken commands in [this slack thread](https://yugabyte.slack.com/archives/CDQB8MEAU/p1616789578068000)

**Comment by deeps1991** [comment]
HI Alex, I added the required commands in the slack thread:

```
Hey Alex, so it seems like we have to rewrite the document structure in some sense. This is because of the additional step to create tablespaces.
I will leave it up to you to make the best decision on where to place the part where tablespaces are actually created, but here are the actual sql statements that are needed to make this work:
1. Table creation:
CREATE TABLE transactions (
    user_id   INTEGER NOT NULL,
    account_id INTEGER NOT NULL,
    geo_partition VARCHAR,
    account_type VARCHAR NOT NULL,
    amount NUMERIC NOT NULL,
    txn_type VARCHAR NOT NULL,
    created_at TIMESTAMP DEFAULT NOW()
) PARTITION BY LIST (geo_partition);
2. Create tablespaces pertaining to each region:
CREATE TABLESPACE eu_central_1_tablespace
WITH (replica_placement='{"num_replicas": 3, "placement_blocks":
[{"cloud":"aws","region":"eu-central-1","zone":"eu-central-1a","min_num_replicas":1},
{"cloud":"aws","region":"eu-central-1","zone":"eu-central-1b","min_num_replicas":1},
{"cloud":"aws","region":"eu-central-1","zone":"eu-central-1c","min_num_replicas":1}]}');
CREATE TABLESPACE us_west_2_tablespace
WITH (replica_placement='{"num_replicas": 3, "placement_blocks":
[{"cloud":"aws","region":"us-west-2","zone":"us-west-2a","min_num_replicas":1},
{"cloud":"aws","region":"us-west-2","zone":"us-west-2b","min_num_replicas":1},
{"cloud":"aws","region":"us-west-2","zone":"us-west-2c","min_num_replicas":1}]}');
CREATE TABLESPACE ap_south_1_tablespace
WITH (replica_placement='{"num_replicas": 3, "placement_blocks":
[{"cloud":"aws","region":"ap-south-1","zone":"ap-south-1a","min_num_replicas":1},
{"cloud":"aws","region":"ap-south-1","zone":"ap-south-1b","min_num_replicas":1},
{"cloud":"aws","region":"ap-south-1","zone":"ap-south-1c","min_num_replicas":1}]}');
3. Create partitions for the table, and associate each partition with the tablespace
having the required placment policy:
CREATE TABLE transactions_eu
    PARTITION OF transactions
      (user_id, account_id, geo_partition, account_type,
       amount, txn_type, created_at,
       PRIMARY KEY (user_id HASH, account_id, geo_partition))
    FOR VALUES IN ('EU') TABLESPACE eu_central_1_tablespace;
CREATE TABLE transactions_india
    PARTITION OF transactions
      (user_id, account_id, geo_partition, account_type,
       amount, txn_type, created_at,
       PRIMARY KEY (user_id HASH, account_id, geo_partition))
    FOR VALUES IN ('India') TABLESPACE ap_south_1_tablespace;
CREATE TABLE transactions_default
    PARTITION OF transactions
      (user_id, account_id, geo_partition, account_type,
       amount, txn_type, created_at,
       PRIMARY KEY (user_id HASH, account_id, geo_partition))
    DEFAULT TABLESPACE us_west_2_tablespace;
```

**Comment by polarweasel** [comment]
@deeps1991 please take a look at the latest change to make sure it's good (and to make sure we're not missing anything else!)

File: docs/content/latest/explore/multi-region-deployments/row-level-geo-partitioning.md [hunk 1]
```diff
@@ -22,11 +22,11 @@ showAsideToc: true
   </li>
 </ul>
 
-
 Row-level geo-partitioning allows fine-grained control over pinning data in a user table (at a per-row level) to geographic locations, thereby allowing the data residency to be managed at the database level. Use-cases requiring low latency multi-region deployments, transactional consistency semantics and transparently schema change propagation across the regions would benefit from this feature. 
 
 {{< tip title="" >}}
 Geo-partitioning makes it easy for developers to move data closer to users for:
+
```
Comment (deeps1991) [inline docs/content/latest/explore/multi-region-deployments/row-level-geo-partitioning.md:29]: Git won't let me add comments on unmodified lines.... 
This is for line 37: The second step involves creating partitions in the respective geographic locations using tablespaces

File: docs/content/latest/explore/multi-region-deployments/row-level-geo-partitioning.md [hunk 2]
```diff
@@ -48,66 +48,91 @@ Let us look at this feature in the context of a use case. Say that a large but i
...
+    CREATE TABLESPACE ap_south_1_tablespace WITH (
+      replica_placement='{"num_replicas": 3, "placement_blocks":
+      [{"cloud":"aws","region":"ap-south-1","zone":"ap-south-1a","min_num_replicas":1},
+      {"cloud":"aws","region":"ap-south-1","zone":"ap-south-1b","min_num_replicas":1},
+      {"cloud":"aws","region":"ap-south-1","zone":"ap-south-1c","min_num_replicas":1}]}'
+    );
+    ```
+
+1. Next, create one partition per desired geography under the parent table. Here, you create three table partitions: one for the EU region called `transactions_eu`, another for the India region called `transactions_india,` and a third default partition for the rest of the regions called `transactions_default`.
```
Comment (deeps1991) [inline docs/content/latest/explore/multi-region-deployments/row-level-geo-partitioning.md:104]: Also we need to mention that we are creating them in their appropriate geographic locations using tablespaces

File: docs/content/latest/explore/multi-region-deployments/row-level-geo-partitioning.md [hunk 3]
```diff
@@ -48,66 +48,91 @@ Let us look at this feature in the context of a use case. Say that a large but i
...
+
+    CREATE TABLE transactions_default 
+        PARTITION OF transactions
+          (user_id, account_id, geo_partition, account_type, 
+          amount, txn_type, created_at,
+          PRIMARY KEY (user_id HASH, account_id, geo_partition))
+        DEFAULT TABLESPACE us_west_2_tablespace;
+    ```
+
+So far, you've created the partitions, but haven't pinned them to the desired  geographical locations. You'll do this in the next section. Use the `\d` command to view the table and partitions you've created so far.
```
Comment (deeps1991) [inline docs/content/latest/explore/multi-region-deployments/row-level-geo-partitioning.md:129]: Actually we have already pinned them :D Thats what the tablespace part did :)

File: docs/content/latest/explore/multi-region-deployments/row-level-geo-partitioning.md [hunk 4]
```diff
@@ -124,38 +149,31 @@ Now that we have a table with the desired three partitions, the final step is to
 
 ![Row-level geo-partitioning](/images/explore/multi-region-deployments/geo-partitioning-2.png)
 
-
 First, we pin the data of the EU partition `transactions_eu` to live across three zones of the Europe (Frankfurt) region `eu-central-1` as shown below.
 
-
-```
+```sh
```
Comment (deeps1991) [inline docs/content/latest/explore/multi-region-deployments/row-level-geo-partitioning.md:154]: The whole of Step2 is now redundant. These yb-admin commands are not needed anymore.
I guess we will still use this diagram, but that will also need to move into Step 1...
Maybe right after we create tablespaces (and before creating the tables themselves?)

File: docs/content/latest/explore/multi-region-deployments/row-level-geo-partitioning.md [hunk 5]
```diff
@@ -330,35 +368,35 @@ txn_type      | debit
...
+```sql
 CREATE TABLE transactions_brazil 
     PARTITION OF transactions
       (user_id, account_id, geo_partition, account_type, 
        amount, txn_type, created_at,
        PRIMARY KEY (user_id HASH, account_id, geo_partition))
     FOR VALUES IN ('Brazil');
+```
 
+```sh
```
Comment (deeps1991) [inline docs/content/latest/explore/multi-region-deployments/row-level-geo-partitioning.md:384]: Again, we won't use yb-admin commands anymore. Instead you would have to use the following commands here:
CREATE TABLESPACE sa_east_1_tablespace WITH (
  replica_placement='{"num_replicas": 3, "placement_blocks":
  [{"cloud":"aws","region":"sa-east-1","zone":"sa-east-1a","min_num_replicas":1},
  {"cloud":"aws","region":"sa-east-1","zone":"sa-east-1b","min_num_replicas":1},
  {"cloud":"aws","region":"sa-east-1","zone":"sa-east-1c","min_num_replicas":1}]}'
);


CREATE TABLE transactions_brazil 
     PARTITION OF transactions
       (user_id, account_id, geo_partition, account_type, 
        amount, txn_type, created_at,
        PRIMARY KEY (user_id HASH, account_id, geo_partition))
     FOR VALUES IN ('Brazil') TABLESPACE sa_east_1_tablespace;

**Review by deeps1991 — CHANGES_REQUESTED** [review]

File: docs/content/latest/explore/multi-region-deployments/row-level-geo-partitioning.md (hunk 1 above)
Comment (polarweasel) [reply docs/content/latest/explore/multi-region-deployments/row-level-geo-partitioning.md:29]: Done

File: docs/content/latest/explore/multi-region-deployments/row-level-geo-partitioning.md (hunk 2 above)
Comment (polarweasel) [reply docs/content/latest/explore/multi-region-deployments/row-level-geo-partitioning.md:104]: Done

File: docs/content/latest/explore/multi-region-deployments/row-level-geo-partitioning.md (hunk 5 above)
Comment (polarweasel) [reply docs/content/latest/explore/multi-region-deployments/row-level-geo-partitioning.md]: updated

File: docs/content/latest/explore/multi-region-deployments/row-level-geo-partitioning.md (hunk 3 above)
Comment (polarweasel) [reply docs/content/latest/explore/multi-region-deployments/row-level-geo-partitioning.md]: Fixed!

File: docs/content/latest/explore/multi-region-deployments/row-level-geo-partitioning.md (hunk 4 above)
Comment (polarweasel) [reply docs/content/latest/explore/multi-region-deployments/row-level-geo-partitioning.md]: And it's gone :)

File: docs/content/latest/explore/multi-region-deployments/row-level-geo-partitioning.md [hunk 6]
```diff
@@ -22,19 +22,19 @@ showAsideToc: true
   </li>
 </ul>
 
-
 Row-level geo-partitioning allows fine-grained control over pinning data in a user table (at a per-row level) to geographic locations, thereby allowing the data residency to be managed at the database level. Use-cases requiring low latency multi-region deployments, transactional consistency semantics and transparently schema change propagation across the regions would benefit from this feature.
```
Comment (deeps1991) [inline docs/content/latest/explore/multi-region-deployments/row-level-geo-partitioning.md:25]: The first line is a bit confusing... We say row level geo partitioning and then say data residency managed at database level... Do you know what we mean to say here?

Comment (deeps1991) [inline docs/content/latest/explore/multi-region-deployments/row-level-geo-partitioning.md:25]: transparently--->transparent?? Not sure, upto you.

**Review by deeps1991 — APPROVED** [review]
@polarweasel Sorry about the delay, I somehow missed this review. It looks good!! Added 2 minor comments/questions but those are up to you

**Comment by polarweasel** [comment]
> The first line is a bit confusing... We say row level geo partitioning and then say data residency managed at database level... Do you know what we mean to say here? 

I think what we mean to say is "managed at the table-row level".

**Pull request merged**